pages/docs/guidelines/project-closure.mdx

```diff
@@ -0,0 +1,72 @@
+---
+title: Project Closure Checklists
+section: Guidelines
+order: 6
+---
+
+# Project Closure Checklists
+While closing a project in truemark, here are the list of recommended actions that one should do:
+
+### Project Retrospective and Feedback Collection: Internal Retrospectives and Client, AM & Team Members feedback 
+- Project Retrospectives complete (internal retros).
+- Ask the HR manager to send out a form to be filled up for Client & AM(Account Manager) and team members feedback.
+
+### Comprehensive Project’s Technical Documentation, Outstanding Tasks and Project Closure Report Compilation
+- Technical documentation in the platform of choice (eg:confluence, GitHub/GitLab)
+- Credentials for hosting and domain stored.
+- Mention what’s still left to do like bugs, requirements etc.
```

Comment:
some lines are ending with . while some are not..please consult the repo for common practice and stick with one behaviour